### DISCUSSION TITLE
Trying to update my laptop when all my changes were done on my desktop

### DISCUSSION THREAD

**Discussion by ilan** [community/community#22582] [New to GitHub]
<p>I did my initial clone on my laptop to get an initial feel for the Maven project on Netbeans.</p>
<p>From there I switched to my desktop with another clone.</p>
<p>I continued to work on my desktop with several commits.</p>
<p>I wanted to look again from my laptop. The only thing I had changed was the pom file, adding a local dependency. The files in the local repository were unchanged, so there was nothing to commit.</p>
<p>Even now my laptop claims “On branch “dev”: in sync with upstream branch”. It depends upon how you look at the situation if the claim is true or not. Since I haven’t changed anything on the laptop, it is true, but I have changed things on the desktop.</p>
<p>If I go Git -&gt; Remote -&gt; Pull from Upstream no change will be made to my source files.</p>
<p>What I would like see to happen is that they all get updated according to the changes I committed from my desktop, but I am missing something to make this happen.</p>
<p>Thanks,</p>
<p>Ilan</p>

**Comment by Wabri** [comment, marked as answer]
<p>Do you have push to remote the commits you made on desktop? If the answer is no so you have to add-commit-push the files to remote. In case you already did, the next question is: the branch on desktop where you made commits, are the same of the branch on laptop that you are trying to pull from remote?</p>

**Comment by ilan** [comment]
<p>Thanks for your reply. I just discovered on my desktop (which is up to date) I am using the master branch. On my laptop I am using a “dev” branch, which I never changed.</p>
<p>I have to figure out how to change Netbeans to use the master branch instead of dev (on the laptop). Then it should update and finally make some sense. When I look at the Git browser both computers are on the master branch.</p>
